src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -63,10 +82,50 @@ public boolean init(Expression<?>[] exprs,
 						ParseResult parseResult,
 						SectionNode sectionNode,
 						List<TriggerItem> triggerItems) {
-		type = ConditionalType.values()[matchedPattern];
-		parseIf = parseResult.mark == 1;
+		type = CONDITIONAL_PATTERNS.getInfo(matchedPattern);
+		ifAny = parseResult.hasTag("any");
+		parseIf = parseResult.hasTag("parse");
+		multiline = parseResult.regexes.size() == 0 && type != ConditionalType.ELSE;
+		boolean explicit = parseResult.hasTag("inline");
```

Comment:
tbh this is only used once so I'm not sure we actually need to declare it as a variable